packages/config/tests/parse/snapshots/tests.js.md

```diff
@@ -2114,6 +2120,14 @@ Generated by [AVA](https://ava.li).
             "query": {},␊
             "to": "/external/path"␊
           },␊
+          {␊
+            "conditions": {},␊
+            "force": false,␊
+            "from": "/external/path",␊
+            "headers": {},␊
+            "query": {},␊
+            "to": "/external/path"␊
+          },␊
```

Comment:
Update 2 tests that load both `_redirects`/`_headers` and `netlify.toml` and ensure that the merged result contains both, but without any duplicate items.
Unfortunately, some file paths get normalized as `/external/path`.